contrib/endpoints/src/api_manager/request_handler.cc

```diff
@@ -30,28 +30,34 @@ namespace google {
 namespace api_manager {
 
 void RequestHandler::Check(std::function<void(Status status)> continuation) {
-  if (context_) {
+  if (api_manager_->ConfigLoadingStatus().ok()) {
```

Comment:
How about aborted?

How about define a function in api_manager_impl as 

bool InitConfigLoadingDone()